cluster-up/cluster/kind-k8s-1.17.0-ipv6/provider.sh

```diff
@@ -11,4 +13,24 @@ source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
 function up() {
     cp $KIND_MANIFESTS_DIR/kind-ipv6.yaml ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/kind.yaml
     kind_up
+    # remove the rancher.io kind default storageClass
+    _kubectl delete sc standard
+
+    nodes=$(_kubectl get nodes -o=custom-columns=:.metadata.name | awk NF)
+    for node in $nodes; do
+        # Create local-volume directories
+        for i in {1..10}; do
```

Comment:
why 10 ?

Reply:
10 is basically a number that seemed large enough to give enough PVs for most tests at the time. With the ability to add more later, it seemed good enough. I still haven't found any cases where 10 was not more than enough.

Reply:
Can you add q comment about it ?

Reply:
Can you add q comment about it ?

Reply:
Can you add q comment about it ?

Reply:
This is actually public facing information, available in the [kubevirtci storage README](https://github.com/kubevirt/kubevirtci/blob/master/cluster-provision/STORAGE.md#storage-providers-inside-the-generated-images) - "By default, we have pre-provisioned 10 local volumes per node, which are available to make requests against. You can manually bind mount more directories to increase this number.".

Added that link to the already existing comment, hope that's enough.

Reply:
good